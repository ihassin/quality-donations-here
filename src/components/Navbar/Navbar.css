#keyword-search-input {
    padding: 8px;
    width: 25rem;
    margin-left:20px; 
}

#cityZip-search-input {
    width:15rem;
}

#donate-title {
    margin: 0 150px; 
    text-transform: uppercase;
    color:white;
    font-size: 24px;
    font-weight: bolder;
}
.form-rounded {
    border-radius: 1rem;
}

div.dropdown-item {
    padding:0px;
}

.dropdown-menu>li>a {
    font-weight: bolder;
    font-size: 14px;
    text-transform: uppercase;
    color:black;
}

nav.navbar.navbar-default {
    margin-bottom: 0px;
    background: #f9a825;
    border:none;
}

.navbar-default .navbar-nav>li>a {
    color:white;
    font-weight: bolder;
    font-size: 14px;
    text-transform: uppercase;
}



