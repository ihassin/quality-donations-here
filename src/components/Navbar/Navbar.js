import React from 'react'
// import {
//   Navbar,
//   Nav,
//   NavItem,
//   NavDropdown,
//   MenuItem,
//   FormGroup,
//   FormControl,
//   ControlLabel
// } from 'react-bootstrap'
import './Navbar.css'
import { Link } from 'react-router-dom'
import LoginM from '../Login/LoginM'
import SignupM from '../Login/SignupM'

const DonateNavbar = props => {
  if (props.loggedIn) {
    return (
      <nav className="navbar navbar-expand-md navbar-light p-0 m-0">
        <div className="container-fluid mx-2 my-2">
          <a className="navbar-brand p-0 mr-4 my-0" href="#">
            <img
              id="logo"
              src={require('../../images/qualitydonationslogo.png')}
            />
          </a>

          <button
            className="navbar-toggler"
            data-toggle="collapse"
            data-target="#navbarNavForm"
          >
            <span className="navbar-toggler-icon" />
          </button>

          <div className="collapse navbar-collapse" id="navbarNavForm">
            {props.shop ? (
              <form className="form-inline">
                <input
                  type="text"
                  className="form-control mr-1"
                  id="keyword-search-input"
                  name="keyword"
                  onChange={props.handleKeywordSearch}
                  placeholder="What are you looking for?"
                />
                <input
                  type="text"
                  className="form-control mr-1"
                  id="cityZip-search-input"
                  name="keyword"
                  onChange={props.handleCityZipSearch}
                  placeholder="Enter City or Zip"
                />
              </form>
            ) : (
              ''
            )}

            {props.donate ? (
              <form className="form" id="donate-title">
                  Donate
              </form>
            ) : (
              ''
            )}

            <ul className="navbar-nav ml-auto">
              {props.shop ? (
                ''
              ) : (
                <li className="nav-item">
                  <a className="nav-link" href="/">
                    shop
                  </a>
                </li>
              )}

              {props.donate ? (
                ''
              ) : (
                <li className="nav-item">
                  <Link className="nav-link" id="donate-link" to="/donate">
                    donate
                  </Link>
                </li>
              )}

              <li className="nav-item dropdown">
                <a
                  className="nav-link dropdown-toggle rounded-circle"
                  data-toggle="dropdown"
                  href="#"
                >
                  {props.user.local.username}
                </a>
                <div className="dropdown-menu">
                  <a href="#" className="dropdown-item">
                    My Donations
                  </a>
                  <a href="#" className="dropdown-item">
                    My Pickups
                  </a>
                  <a href="#" className="dropdown-item">
                    My Profile
                  </a>
                </div>
              </li>

              <li className="nav-item">
                <a className="nav-link" href="#" onClick={props._logout}>
                  logout
                </a>
              </li>
            </ul>
          </div>
        </div>
      </nav>
    )
  } else {
    return (
      <nav className="navbar navbar-expand-md navbar-light p-0 m-0">
        <div className="container-fluid mx-2 my-2">
          <a className="navbar-brand p-0 mr-4 my-0" href="#">
            <img
              id="logo"
              src={require('../../images/qualitydonationslogo.png')}
            />
          </a>

          <button
            className="navbar-toggler"
            data-toggle="collapse"
            data-target="#navbarNavForm"
          >
            <span className="navbar-toggler-icon" />
          </button>

          <div className="collapse navbar-collapse" id="navbarNavForm">
            {props.shop ? (
              <form className="form-inline mr-1">
                <input
                  type="text"
                  className="form-control mr-sm-2"
                  id="keyword-search-input"
                  name="keyword"
                  onChange={props.handleKeywordSearch}
                  placeholder="What are you looking for?"
                />
                <input
                  type="text"
                  className="form-control mr-1"
                  id="cityZip-search-input"
                  name="keyword"
                  onChange={props.handleCityZipSearch}
                  placeholder="Enter City or Zip"
                />
              </form>
            ) : (
              ''
            )}
            <ul className="navbar-nav ml-auto">
              <li className="nav-item">
                <a
                  className="nav-link"
                  href="#"
                  // onClick={props.handleShowLogIn}
                  data-toggle="modal"
                  data-target="#loginModal"
                >
                  donate
                </a>
              </li>

              <li className="nav-item">
                <a
                  className="nav-link"
                  href="#"
                  // onClick={props.handleShowLogIn}
                  data-toggle="modal"
                  data-target="#loginModal"
                >
                  login
                </a>
              </li>

              <li className="nav-item">
                <a
                  className="nav-link"
                  href="#"
                  // onClick={props.handleShowSignup}
                  data-toggle="modal"
                  data-target="#signupModal"
                >
                  sign up
                </a>
              </li>
            </ul>
          </div>
        </div>
        <LoginM
          username={props.username}
          password={props.password}
          handleChange={props.handleChange}
          handleLoginSubmit={props.handleLoginSubmit}
        />

        <SignupM
          signupFirstname={props.signupFirstname}
          signupLastname={props.signupLastname}
          signupEmail={props.signupEmail}
          signupUsername={props.signupUsername}
          signupPassword={props.signupPassword}
          signupconfirmPassword={props.signupconfirmPassword}
          handleChange={props.handleChange}
          handleSignupSubmit={props.handleSignupSubmit}
        />
      </nav>
    )
  }
}

export default DonateNavbar
