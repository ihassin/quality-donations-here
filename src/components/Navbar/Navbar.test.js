import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './Navbar';

const navbar = shallow(<Navbar />);

it('renders correctly', () => {
    expect(navbar).toMatchSnapshot();
})
