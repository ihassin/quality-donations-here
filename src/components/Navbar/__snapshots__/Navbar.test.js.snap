// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DonateNavbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="container-fluid mx-2 my-2"
>
          <a
                    className="navbar-brand p-0 mr-4 my-0"
                    href="#"
          >
                    <img
                              id="logo"
                              src="qualitydonationslogo.png"
                    />
          </a>
          <button
                    className="navbar-toggler"
                    data-target="#navbarNavForm"
                    data-toggle="collapse"
          >
                    <span
                              className="navbar-toggler-icon"
                    />
          </button>
          <div
                    className="collapse navbar-collapse"
                    id="navbarNavForm"
          >
                    
                    <ul
                              className="navbar-nav ml-auto"
                    >
                              <li
                                        className="nav-item"
                              >
                                        <a
                                                  className="nav-link"
                                                  data-target="#loginModal"
                                                  data-toggle="modal"
                                                  href="#"
                                        >
                                                  donate
                                        </a>
                              </li>
                              <li
                                        className="nav-item"
                              >
                                        <a
                                                  className="nav-link"
                                                  data-target="#loginModal"
                                                  data-toggle="modal"
                                                  href="#"
                                        >
                                                  login
                                        </a>
                              </li>
                              <li
                                        className="nav-item"
                              >
                                        <a
                                                  className="nav-link"
                                                  data-target="#signupModal"
                                                  data-toggle="modal"
                                                  href="#"
                                        >
                                                  sign up
                                        </a>
                              </li>
                    </ul>
          </div>
</div>,
        <LoginM
          handleChange={undefined}
          handleLoginSubmit={undefined}
          password={undefined}
          username={undefined}
/>,
        <SignupM
          handleChange={undefined}
          handleSignupSubmit={undefined}
          signupEmail={undefined}
          signupFirstname={undefined}
          signupLastname={undefined}
          signupPassword={undefined}
          signupUsername={undefined}
          signupconfirmPassword={undefined}
/>,
      ],
      "className": "navbar navbar-expand-md p-0 m-0",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <a
              className="navbar-brand p-0 mr-4 my-0"
              href="#"
>
              <img
                            id="logo"
                            src="qualitydonationslogo.png"
              />
</a>,
            <button
              className="navbar-toggler"
              data-target="#navbarNavForm"
              data-toggle="collapse"
>
              <span
                            className="navbar-toggler-icon"
              />
</button>,
            <div
              className="collapse navbar-collapse"
              id="navbarNavForm"
>
              
              <ul
                            className="navbar-nav ml-auto"
              >
                            <li
                                          className="nav-item"
                            >
                                          <a
                                                        className="nav-link"
                                                        data-target="#loginModal"
                                                        data-toggle="modal"
                                                        href="#"
                                          >
                                                        donate
                                          </a>
                            </li>
                            <li
                                          className="nav-item"
                            >
                                          <a
                                                        className="nav-link"
                                                        data-target="#loginModal"
                                                        data-toggle="modal"
                                                        href="#"
                                          >
                                                        login
                                          </a>
                            </li>
                            <li
                                          className="nav-item"
                            >
                                          <a
                                                        className="nav-link"
                                                        data-target="#signupModal"
                                                        data-toggle="modal"
                                                        href="#"
                                          >
                                                        sign up
                                          </a>
                            </li>
              </ul>
</div>,
          ],
          "className": "container-fluid mx-2 my-2",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                id="logo"
                src="qualitydonationslogo.png"
/>,
              "className": "navbar-brand p-0 mr-4 my-0",
              "href": "#",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "id": "logo",
                "src": "qualitydonationslogo.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <span
                className="navbar-toggler-icon"
/>,
              "className": "navbar-toggler",
              "data-target": "#navbarNavForm",
              "data-toggle": "collapse",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "navbar-toggler-icon",
              },
              "ref": null,
              "rendered": null,
              "type": "span",
            },
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "",
                <ul
                  className="navbar-nav ml-auto"
>
                  <li
                                    className="nav-item"
                  >
                                    <a
                                                      className="nav-link"
                                                      data-target="#loginModal"
                                                      data-toggle="modal"
                                                      href="#"
                                    >
                                                      donate
                                    </a>
                  </li>
                  <li
                                    className="nav-item"
                  >
                                    <a
                                                      className="nav-link"
                                                      data-target="#loginModal"
                                                      data-toggle="modal"
                                                      href="#"
                                    >
                                                      login
                                    </a>
                  </li>
                  <li
                                    className="nav-item"
                  >
                                    <a
                                                      className="nav-link"
                                                      data-target="#signupModal"
                                                      data-toggle="modal"
                                                      href="#"
                                    >
                                                      sign up
                                    </a>
                  </li>
</ul>,
              ],
              "className": "collapse navbar-collapse",
              "id": "navbarNavForm",
            },
            "ref": null,
            "rendered": Array [
              "",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li
                      className="nav-item"
>
                      <a
                                            className="nav-link"
                                            data-target="#loginModal"
                                            data-toggle="modal"
                                            href="#"
                      >
                                            donate
                      </a>
</li>,
                    <li
                      className="nav-item"
>
                      <a
                                            className="nav-link"
                                            data-target="#loginModal"
                                            data-toggle="modal"
                                            href="#"
                      >
                                            login
                      </a>
</li>,
                    <li
                      className="nav-item"
>
                      <a
                                            className="nav-link"
                                            data-target="#signupModal"
                                            data-toggle="modal"
                                            href="#"
                      >
                                            sign up
                      </a>
</li>,
                  ],
                  "className": "navbar-nav ml-auto",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        data-target="#loginModal"
                        data-toggle="modal"
                        href="#"
>
                        donate
</a>,
                      "className": "nav-item",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "donate",
                        "className": "nav-link",
                        "data-target": "#loginModal",
                        "data-toggle": "modal",
                        "href": "#",
                      },
                      "ref": null,
                      "rendered": "donate",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        data-target="#loginModal"
                        data-toggle="modal"
                        href="#"
>
                        login
</a>,
                      "className": "nav-item",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "login",
                        "className": "nav-link",
                        "data-target": "#loginModal",
                        "data-toggle": "modal",
                        "href": "#",
                      },
                      "ref": null,
                      "rendered": "login",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        data-target="#signupModal"
                        data-toggle="modal"
                        href="#"
>
                        sign up
</a>,
                      "className": "nav-item",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "sign up",
                        "className": "nav-link",
                        "data-target": "#signupModal",
                        "data-toggle": "modal",
                        "href": "#",
                      },
                      "ref": null,
                      "rendered": "sign up",
                      "type": "a",
                    },
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "handleChange": undefined,
          "handleLoginSubmit": undefined,
          "password": undefined,
          "username": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "handleChange": undefined,
          "handleSignupSubmit": undefined,
          "signupEmail": undefined,
          "signupFirstname": undefined,
          "signupLastname": undefined,
          "signupPassword": undefined,
          "signupUsername": undefined,
          "signupconfirmPassword": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="container-fluid mx-2 my-2"
>
            <a
                        className="navbar-brand p-0 mr-4 my-0"
                        href="#"
            >
                        <img
                                    id="logo"
                                    src="qualitydonationslogo.png"
                        />
            </a>
            <button
                        className="navbar-toggler"
                        data-target="#navbarNavForm"
                        data-toggle="collapse"
            >
                        <span
                                    className="navbar-toggler-icon"
                        />
            </button>
            <div
                        className="collapse navbar-collapse"
                        id="navbarNavForm"
            >
                        
                        <ul
                                    className="navbar-nav ml-auto"
                        >
                                    <li
                                                className="nav-item"
                                    >
                                                <a
                                                            className="nav-link"
                                                            data-target="#loginModal"
                                                            data-toggle="modal"
                                                            href="#"
                                                >
                                                            donate
                                                </a>
                                    </li>
                                    <li
                                                className="nav-item"
                                    >
                                                <a
                                                            className="nav-link"
                                                            data-target="#loginModal"
                                                            data-toggle="modal"
                                                            href="#"
                                                >
                                                            login
                                                </a>
                                    </li>
                                    <li
                                                className="nav-item"
                                    >
                                                <a
                                                            className="nav-link"
                                                            data-target="#signupModal"
                                                            data-toggle="modal"
                                                            href="#"
                                                >
                                                            sign up
                                                </a>
                                    </li>
                        </ul>
            </div>
</div>,
          <LoginM
            handleChange={undefined}
            handleLoginSubmit={undefined}
            password={undefined}
            username={undefined}
/>,
          <SignupM
            handleChange={undefined}
            handleSignupSubmit={undefined}
            signupEmail={undefined}
            signupFirstname={undefined}
            signupLastname={undefined}
            signupPassword={undefined}
            signupUsername={undefined}
            signupconfirmPassword={undefined}
/>,
        ],
        "className": "navbar navbar-expand-md p-0 m-0",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <a
                className="navbar-brand p-0 mr-4 my-0"
                href="#"
>
                <img
                                id="logo"
                                src="qualitydonationslogo.png"
                />
</a>,
              <button
                className="navbar-toggler"
                data-target="#navbarNavForm"
                data-toggle="collapse"
>
                <span
                                className="navbar-toggler-icon"
                />
</button>,
              <div
                className="collapse navbar-collapse"
                id="navbarNavForm"
>
                
                <ul
                                className="navbar-nav ml-auto"
                >
                                <li
                                                className="nav-item"
                                >
                                                <a
                                                                className="nav-link"
                                                                data-target="#loginModal"
                                                                data-toggle="modal"
                                                                href="#"
                                                >
                                                                donate
                                                </a>
                                </li>
                                <li
                                                className="nav-item"
                                >
                                                <a
                                                                className="nav-link"
                                                                data-target="#loginModal"
                                                                data-toggle="modal"
                                                                href="#"
                                                >
                                                                login
                                                </a>
                                </li>
                                <li
                                                className="nav-item"
                                >
                                                <a
                                                                className="nav-link"
                                                                data-target="#signupModal"
                                                                data-toggle="modal"
                                                                href="#"
                                                >
                                                                sign up
                                                </a>
                                </li>
                </ul>
</div>,
            ],
            "className": "container-fluid mx-2 my-2",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  id="logo"
                  src="qualitydonationslogo.png"
/>,
                "className": "navbar-brand p-0 mr-4 my-0",
                "href": "#",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "id": "logo",
                  "src": "qualitydonationslogo.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <span
                  className="navbar-toggler-icon"
/>,
                "className": "navbar-toggler",
                "data-target": "#navbarNavForm",
                "data-toggle": "collapse",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "navbar-toggler-icon",
                },
                "ref": null,
                "rendered": null,
                "type": "span",
              },
              "type": "button",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "",
                  <ul
                    className="navbar-nav ml-auto"
>
                    <li
                                        className="nav-item"
                    >
                                        <a
                                                            className="nav-link"
                                                            data-target="#loginModal"
                                                            data-toggle="modal"
                                                            href="#"
                                        >
                                                            donate
                                        </a>
                    </li>
                    <li
                                        className="nav-item"
                    >
                                        <a
                                                            className="nav-link"
                                                            data-target="#loginModal"
                                                            data-toggle="modal"
                                                            href="#"
                                        >
                                                            login
                                        </a>
                    </li>
                    <li
                                        className="nav-item"
                    >
                                        <a
                                                            className="nav-link"
                                                            data-target="#signupModal"
                                                            data-toggle="modal"
                                                            href="#"
                                        >
                                                            sign up
                                        </a>
                    </li>
</ul>,
                ],
                "className": "collapse navbar-collapse",
                "id": "navbarNavForm",
              },
              "ref": null,
              "rendered": Array [
                "",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <li
                        className="nav-item"
>
                        <a
                                                className="nav-link"
                                                data-target="#loginModal"
                                                data-toggle="modal"
                                                href="#"
                        >
                                                donate
                        </a>
</li>,
                      <li
                        className="nav-item"
>
                        <a
                                                className="nav-link"
                                                data-target="#loginModal"
                                                data-toggle="modal"
                                                href="#"
                        >
                                                login
                        </a>
</li>,
                      <li
                        className="nav-item"
>
                        <a
                                                className="nav-link"
                                                data-target="#signupModal"
                                                data-toggle="modal"
                                                href="#"
                        >
                                                sign up
                        </a>
</li>,
                    ],
                    "className": "navbar-nav ml-auto",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <a
                          className="nav-link"
                          data-target="#loginModal"
                          data-toggle="modal"
                          href="#"
>
                          donate
</a>,
                        "className": "nav-item",
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "donate",
                          "className": "nav-link",
                          "data-target": "#loginModal",
                          "data-toggle": "modal",
                          "href": "#",
                        },
                        "ref": null,
                        "rendered": "donate",
                        "type": "a",
                      },
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <a
                          className="nav-link"
                          data-target="#loginModal"
                          data-toggle="modal"
                          href="#"
>
                          login
</a>,
                        "className": "nav-item",
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "login",
                          "className": "nav-link",
                          "data-target": "#loginModal",
                          "data-toggle": "modal",
                          "href": "#",
                        },
                        "ref": null,
                        "rendered": "login",
                        "type": "a",
                      },
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": <a
                          className="nav-link"
                          data-target="#signupModal"
                          data-toggle="modal"
                          href="#"
>
                          sign up
</a>,
                        "className": "nav-item",
                      },
                      "ref": null,
                      "rendered": Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "sign up",
                          "className": "nav-link",
                          "data-target": "#signupModal",
                          "data-toggle": "modal",
                          "href": "#",
                        },
                        "ref": null,
                        "rendered": "sign up",
                        "type": "a",
                      },
                      "type": "li",
                    },
                  ],
                  "type": "ul",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "handleChange": undefined,
            "handleLoginSubmit": undefined,
            "password": undefined,
            "username": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "handleChange": undefined,
            "handleSignupSubmit": undefined,
            "signupEmail": undefined,
            "signupFirstname": undefined,
            "signupLastname": undefined,
            "signupPassword": undefined,
            "signupUsername": undefined,
            "signupconfirmPassword": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
